### PR TITLE
V4.2-dev

### PR DESCRIPTION
**Pull request by DaanSelen**
The new PR!
https://github.com/jpizquierdo
https://github.com/TheSL18

### CODE REVIEW

File: src/static/locale/active_languages.json
Comment (DaanSelen) [inline src/static/locale/active_languages.json:1]: Can you make it Alphabetical? Then we can Merge!

Comment (DaanSelen) [reply src/static/locale/active_languages.json:1]: Make sure to keep capitalization like the others :)

**Comment by jpizquierdo** [comment]
Thanks, i think that you forgot to mention is with de "@", so I have just read because I ipen the repo :)

 @TheSL18 , just to let you know

**Comment by DaanSelen** [comment]
> Thanks, i think that you forgot to mention is with de "@", so I have just read because I ipen the repo :)
> 
> @TheSL18 , just to let you know that there are some small fixes to make in @DaanSelen comments

Thanks! Once those are ironed out and you approve, we can merge into 4.2

**Review by jpizquierdo — APPROVED** [review]

File: src/static/locale/active_languages.json [hunk 1]
```diff
@@ -1,47 +1,58 @@
...
+    "lang_id": "es",
+    "lang_name": "Spanish",
+    "lang_name_localized": "Español"
+  },
+  {
+    "lang_id": "it-it",
+    "lang_name": "Italian",
+    "lang_name_localized": "Italiano"
+  },
+  {
```
Comment (jpizquierdo) [inline src/static/locale/active_languages.json:27]: italian is duplicated, the second one seem to be needed to remove

File: src/static/locale/active_languages.json [hunk 2]
```diff
@@ -1,47 +1,58 @@
...
+    "lang_id": "zh-cn",
+    "lang_name": "Chinese (Simplified)",
+    "lang_name_localized": "中文（简体）"
+  },
+  {
+    "lang_id": "zh-hk",
+    "lang_name": "Chinese (Traditional)",
+    "lang_name_localized": "中文（繁體）"
+  },
+  {
```
Comment (jpizquierdo) [inline src/static/locale/active_languages.json:52]: "uk" goes after "ru"

**Review by jpizquierdo — CHANGES_REQUESTED** [review]
there are some fixes needed, commented in the lines affected

**Comment by DaanSelen** [comment]
You can always provide additions to the Spanish or other translations!

**Pull request closed without merging**